### ISSUE TITLE
Batch Normalization issues

### ISSUE THREAD

**Issue by KevinHexin** [ankurhanda/tf-unet#3]
hello, have you test your trained model? i saved the ckpt files after training, then restore ckpt file to test. it seems like the model can't save moving average of mean and variance in BN. if i set 
layers.batch_norm() para "is_training" to False, the test result will be very bad. i used several bn implements like tf.layers.batch_norm() and tf.nn.batch_normalization() but all can't work.
thanks.

**Comment by KevinHexin** [comment]
I solved this problem. befor train each epoch, don't forget shuffle the data.
